docs/admin/uninstall.md

```diff
@@ -0,0 +1,17 @@
+# Uninstall the App from Nautobot
+
+Here you will find any steps necessary to cleanly remove the App from your Nautobot environment.
+
+## Uninstall Guide
```

Comment:
I think we should do a "write once" guide in Core for this.

Any one-offs can be noted here.